### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by jKwibe**
Add links to add and deduct items in the cart.

### CODE REVIEW

**Comment by jKwibe** [comment]
Pull request for user story 23 and 24

File: Gemfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ group :development, :test do
   gem 'pry'
   gem 'simplecov'
   gem 'shoulda-matchers'
+  gem 'table_print', '~> 1.5', '>= 1.5.7'
```
Comment (RyN21) [inline Gemfile:27]: cool gem!

File: app/views/cart/show.html.erb [hunk 2]
```diff
@@ -30,7 +30,9 @@
       </td>
       <td>
         <center>
+          <p><%= link_to_if quantity < item.inventory, '+', "/cart/#{item.id}?type=add", method: :patch %></p>
```
Comment (RyN21) [inline app/views/cart/show.html.erb:33]: link_to_if... I like that

File: spec/models/cart_spec.rb [hunk 3]
```diff
@@ -31,6 +31,16 @@
         })
     end
 
+    it '.remove_item()' do
```
Comment (RyN21) [inline spec/models/cart_spec.rb:34]: nice test for the model

File: app/controllers/cart_controller.rb [hunk 4]
```diff
@@ -21,6 +21,18 @@ def remove_item
     redirect_to '/cart'
   end
 
+  def update
```
Comment (RyN21) [inline app/controllers/cart_controller.rb:24]: looks good!

File: Gemfile (hunk 1 above)
Comment (sciencefixion) [reply Gemfile:27]: This is going to be great!

**Pull request merged**